include/seqan3/range/view/take_line.hpp

```diff
@@ -427,10 +70,8 @@ namespace seqan3::view
  *
  * On single pass std::ranges::InputRange it can be used to tokenise the input stream line-wise:
  * \snippet test/snippet/range/view/take_line.cpp tokenise
- *
- * \hideinitializer
  */
-inline auto constexpr take_line = detail::take_line_fn<false>{};
+inline auto constexpr take_line = view::take_until_and_consume(is_char<'\r'> || is_char<'\n'>);
```

Comment:
Are you sure that this consumes `\r\n` fully and not just `\r` in case of windows line endings?